src/app/fdctl/config/default.toml

```diff
@@ -1,585 +1,663 @@
     # `--account-index-exclude-key` argument.
     account_index_exclude_keys = []
 
-    # Whether to use compression when storing snapshots. This option is passed
-    # to the Solana Labs client with the `--snapshot-archive-format` argument.
+    # Whether to use compression when storing snapshots.  This option is
+    # passed to the Solana Labs client with the
+    # `--snapshot-archive-format` argument.
     snapshot_archive_format = "zstd"
 
-    # Refuse to start if saved tower state is not found. This option is passed
-    # to the Solana Labs client with the `--require-tower` argument.
+    # Refuse to start if saved tower state is not found.  This option is
+    # passed to the Solana Labs client with the `--require-tower`
+    # argument.
     require_tower = false
 
 [gossip]
-    # Routable DNS name or IP address and port number to use to rendezvous with
-    # the gossip cluster. These entrypoints are passed to the Solana Labs client
-    # with the `--entrypoint` argument.
+    # Routable DNS name or IP address and port number to use to
+    # rendezvous with the gossip cluster.  These entrypoints are passed
+    # to the Solana Labs client with the `--entrypoint` argument.
     entrypoints = []
 
-    # If true, on startup the validator checks that it can connect to all of the
-    # entrypoints on the given port. If not true, the validator will exit. This
-    # option is passed to the Solana Labs client inverted with the
-    # `--no-port-check` argument.
+    # If true, on startup the validator checks that it can connect to
+    # all of the entrypoints on the given port.  If not true, the
+    # validator will exit.  This option is passed to the Solana Labs
+    # client inverted with the `--no-port-check` argument.
     port_check = true
 
-    # The port number to use for receiving gossip messages on this validator.
-    # This option is passed to the Solana Labs client with the `--gossip-port`
-    # argument. This argument is required. Solana Labs treats this as an
-    # optional argument, but the code that tries to find a default will always
-    # fail.
+    # The port number to use for receiving gossip messages on this
+    # validator. This option is passed to the Solana Labs client with
+    # the `--gossip-port` argument.  This argument is required.  Solana
+    # Labs treats this as an optional argument, but the code that tries
+    # to find a default will always fail.
     port = 8001
 
-    # DNS name or IP address to advertise to the network in gossip. If none is
-    # provided, the default is to use the first entrypoint, or if none are
-    # specified, to use 127.0.0.1. This option is passed to the Solana Labs
-    # client with the `--gossip-host` argument.
+    # DNS name or IP address to advertise to the network in gossip.  If
+    # none is provided, the default is to use the first entrypoint, or
+    # if none are specified, to use 127.0.0.1.  This option is passed to
```

Comment:
The default is to ask the entrypoint who it got the ping from.  Using the entrypoint's hostname/IP here would make no sense.

>  .help("Gossip DNS name or IP address for the node to advertise in gossip \
                               [default: ask --entrypoint, or 127.0.0.1 when --entrypoint is not provided]"),